### PR TITLE
Corrections to #1207 to fix broken tests

### LINKED ISSUE #1234: Deprecation notice for Jest config
I note the following deprecation notice when running the FDC3 module tests:

```
> @finos/fdc3@2.1.0 test
> jest --verbose

ts-jest[ts-jest-transformer] (WARN) Define `ts-jest` config under `globals` is deprecated. Please do
transform: {
    <transform_regex>: ['ts-jest', { /* ts-jest config goes here in Jest */ }],
},
ts-jest[ts-jest-transformer] (WARN) Define `ts-jest` config under `globals` is deprecated. Please do
transform: {
    <transform_regex>: ['ts-jest', { /* ts-jest config goes here in Jest */ }],
},
ts-jest[ts-jest-transformer] (WARN) Define `ts-jest` config under `globals` is deprecated. Please do
transform: {
    <transform_regex>: ['ts-jest', { /* ts-jest config goes here in Jest */ }],
},
```

https://github.com/finos/FDC3/blob/34147168e369baf67678fd34882d1a4bf88ebd84/jest.config.js#L4-L12

### LINKED ISSUE #1235: addEventListener test failing in main
The addEventListener test added in in #1207 is failing:

```
 ● test ES6 module › without `window.fdc3` global › addEventListener should reject

    TypeError: (0 , src_1.addEventListener) is not a function

      101 |
      102 |     test('addEventListener should reject', async () => {
    > 103 |       await expect(addEventListener(expect.any(String), expect.any(Object))).rejects.toEqual(UnavailableError);
          |                                    ^
      104 |     });
      105 |
      106 |     test('joinChannel should reject', async () => {

      at test/Methods.test.ts:103:36
      at node_modules/tslib/tslib.js:169:75
      at Object.__awaiter (node_modules/tslib/tslib.js:165:16)
      at Object.<anonymous> (test/Methods.test.ts:102:55)

Test Suites: 1 failed, 2 passed, 3 total
Tests:       1 failed, 54 passed, 55 total
```

### PR DESCRIPTION
**Pull request by kriswest**
resolves #1234 by correcting deprecated JEST config 
resolves #1235 by adding missing addEventListener function to Methods.ts

### CODE REVIEW

**Comment by kriswest** [comment]
@bingenito @finos/fdc3-maintainers here's a great example of why the tests should be required to pass before a PR can be merged. Somehow 4 of us looked at this and didn't see the npm-build tests failing! That's probably because they have not run and there was no failure to see:

https://github.com/finos/FDC3/actions/workflows/package.yml?query=is%3Afailure

**Review by hughtroeger — APPROVED** [review]

**Review by openfin-johans — APPROVED** [review]
Ship it captain!

**Review by hughtroeger — APPROVED** [review]

**Review by bingenito — APPROVED** [review]

**Pull request merged**
